internal/network/machine_network_cidr.go

```diff
@@ -86,13 +86,13 @@ func VerifyVip(hosts []*models.Host, machineNetworkCidr string, vip string, vipN
 		return nil
 	}
 	if machineNetworkCidr == "" {
-		return errors.Errorf("%s <%s> cannot be set if Machine Network CIDR is empty", vipName, vip)
+		return errors.Errorf("%s <%s> cannot be set if Machine Network CIDR is empty. Please make sure that Machine Network CIDR is configured correctly", vipName, vip)
 	}
 	if !ipInCidr(vip, machineNetworkCidr) {
-		return errors.Errorf("%s <%s> does not belong to machine-network-cidr <%s>", vipName, vip, machineNetworkCidr)
+		return errors.Errorf("%s <%s> does not belong to machine-network-cidr <%s>. Try to increase the block range.", vipName, vip, machineNetworkCidr)
```

Comment:
Also here, I think the advice should be "pick a VIP that's within the machine network CIDR". Making the CIDR bigger will not necessarily help, e.g. if the VIP is completely foreign to it.

Also machine network is often something that the user has no control over, they can choose a different VIP but sometimes the machine network is outside their control

Reply:
+1

Reply:
scrapping this solution in favor of a range validation